test/endpoints/auth.spec.js

```diff
@@ -9,13 +9,75 @@ const User = mongoose.model('User');
 const request = supertest.agent(app);
 
 describe('Authentication', () => {
-  describe('Signup', () => {
-    it('Should return a JWT upon user signup', () => {
-      should(1).equal(1);
+  describe('Test for signup with an invalid password', () => {
+    it('Should flag invalid password errors', (done) => {
+      request
+        .post('/api/auth/signup')
+        .send({
+          name: 'tajudeen',
+          email: 'were@taju.com',
+          password: 'qq'
+        })
+        .end((err, res) => {
+          should(res.body.errors[0].msg).equal('passwords must be at least 6 chars long and contain one number');
+          should(res.status).equal(422);
+          if (err) return done(err);
+          done();
+        });
     });
   });
 
-  describe('Signin', () => {
-    it('Should be able to login', () => should(2).equal(2));
+  describe('Test for signup with an invalid username', () => {
+    it('Should flag invalid username errors', (done) => {
```

Comment:
Should this be `name` or `username`? @OlatunjiYso

Reply:
@fortunee, It is "name" actually. Thanks.
I have updated the new PR, this PR is already closed. 

@fortunee,
I closed this PR because I started the branch name with "ft" instead of "ch".
sorry for the inconvenience.